install.sh

```diff
@@ -36,8 +38,7 @@ if [ ! -d "$C9_DIR" ]; then
 fi
 
 VERSION=1
-NODE_VERSION=v6.3.1
-NODE_VERSION_ARM_PI=v0.10.28
+NODE_VERSION=v12.16.1
```

Comment:
Node 16 has known security issues—see https://nodejs.org/en/blog/release/v12.20.1/. Can we instead add `12.20.1` to our CDN and use that here?

```suggestion
NODE_VERSION=v12.20.1
```

Reply:
Hmm `12.20.1` has security issues too. Maybe it would be better to use `12.22.x`. `12.22.9` is the version we current install at runtime.

```suggestion
NODE_VERSION=v12.22.9
```